lib/node_modules/@stdlib/lapack/base/dppequ/lib/base.js

```diff
@@ -0,0 +1,126 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var sqrt = require( '@stdlib/math/base/special/sqrt' );
+var max = require( '@stdlib/math/base/special/max' );
+var min = require( '@stdlib/math/base/special/min' );
+
+
+// MAIN //
+
+/**
+* Computes the row and column scalings intended to equilibrate a symmetric positive definite matrix `A` in packed storage and reduce it's condition number (with respect to the two-norm).
+*
+* @param {string} order - specifies whether `AP` is packed in row-major or column-major order
+* @param {string} uplo - 'Upper' or 'Lower' triangle of `A` is stored
+* @param {NonNegativeInteger} N - order of the matrix `A`
+* @param {Float64Array} AP - array containing the upper or lower triangle of `A` in packed form
+* @param {integer} strideAP - stride length for `AP`
+* @param {integer} offsetAP - starting index for `AP`
+* @param {Float64Array} S - array to store the scale factors of `A`
+* @param {integer} strideS - stride length for `S`
+* @param {integer} offsetS - starting index for `S`
+* @param {Float64Array} out - array to store the output
+* @param {integer} strideOut - stride length for `out`
+* @param {integer} offsetOut - starting index for `out`
+* @returns {integer} status code
+*/
+function dppequ( order, uplo, N, AP, strideAP, offsetAP, S, strideS, offsetS, out, strideOut, offsetOut ) { // eslint-disable-line max-len, max-params
+	var info;
+	var smin;
+	var amax;
+	var jj;
+	var i;
+
+	if ( N === 0 ) {
+		out[ offsetOut ] = 1.0; // scond
+		out[ offsetOut + strideOut ] = 0.0; // amax
+		return 0; // info
+	}
+
+	S[ offsetAP ] = AP[ offsetAP ];
+	smin = S[ offsetAP ];
+	amax = S[ offsetAP ];
+
+	if ( order === 'row-major' ) {
+		if ( uplo === 'U' ) { // uplo === 'U'
+			jj = 0;
+			for ( i = 1; i < N; i++ ) {
+				jj += N - i + 1;
+				S[ offsetS + (i * strideS) ] = AP[ offsetAP + (jj * strideAP) ];
+				smin = min( smin, S[ offsetS + (i * strideS) ] );
+				amax = max( amax, S[ offsetS + (i * strideS) ] );
+			}
+		} else { // uplo === 'L'
+			jj = 0;
+			for ( i = 1; i < N; i++ ) {
+				jj += i + 1;
+				S[ offsetS + (i * strideS) ] = AP[ offsetAP + (jj * strideAP) ];
+				smin = min( smin, S[ offsetS + (i * strideS) ] );
+				amax = max( amax, S[ offsetS + (i * strideS) ] );
+			}
+		}
+	} else { // order === 'col-major'
+		if ( uplo === 'U' ) { // uplo === 'U'
```

Comment:
```suggestion
	} else if ( uplo === 'U' ) { // order === 'column-major' && uplo === 'U'
```

The change is trivial.